packages/textfield/src/textfield.css

```diff
@@ -56,7 +56,25 @@ textarea {
     overflow: hidden;
 }
 
+:host([grows]) #textfield:after {
```

Comment:
If we fully move to the Spectrum CSS implementation here, Textfield can be delivered like this: 
<img width="227" alt="image" src="https://user-images.githubusercontent.com/1156657/236907726-a9a8ecf4-a82e-4128-9797-4d4bd9f562df.png">


I'm not sure you can achieve this layout with `box-shadow`. It may be necessary to move to `outline` for the focus loupe and get over the fact that older versions of Safari will receive square corners.

Reply:
@pfulton can you help us on a path forward here. This issue needs to be addressed in some way. `outline` would work and aligns with my preference for progressive enhancement. In a way it could begin a shift towards a simpler focus loupe across the whole library, but it or something like this is a requirement here.

Reply:
I think a move toward `outline` could be ok here, but let me double-check with design folks before I make a final call.

Reply:
By the way, current MacOS Safari is ok with the `border-radius` values. iOS Safari, however, is not.

**MacOS:**

<img width="271" alt="Screen Shot 2023-05-11 at 1 42 12 PM" src="https://github.com/adobe/spectrum-web-components/assets/360251/9082b76d-7c27-4a4b-925d-baed54e16310">

---

**iOS:**

![Simulator Screen Shot - iPhone 14 Pro - 2023-05-11 at 13 41 58](https://github.com/adobe/spectrum-web-components/assets/360251/a6da2f48-076c-4882-a68e-ab986960e0f4)

Reply:
Which version are you on? 
![IMG_5195](https://github.com/adobe/spectrum-web-components/assets/1156657/e981a253-6bbb-417b-9f92-aea273ba1a51)

Reply:
😱 hrmm `16.2` according to the simulator.